File: ALGORITMOS-E-ESTRUTURAS-DE-DADOS-II (PT2)/#Teorico/0 - Revisao/6 - Comandos de Repetição/repeat_comand.java
public class repeat_comand {
    public static void main(String[] args) {
        int area, cont = 1;

        for (int base = 1; base <= 5; base++) {
            for (int altura = 2; altura <= 10; altura += 2, cont++){
                area = base * altura;

                System.out.println(cont+ "º retângulo");
                System.out.println("====================");
                System.out.println("Base = " + base + " e altura = " + altura);
                System.out.println("Área = " + area + "\n");
            }
        }
    }    
}

File: ALGORITMOS-E-ESTRUTURAS-DE-DADOS-II (PT2)/#Teorico/0 - Revisao/6 - Comandos de Repetição/repeat_comand.py
#Exemplo 1
base = float(input("Digite a base do retângulo: "))

while base > 0:
    altura = float(input("Digite a altura do retângulo: "))
    
    area = base * altura
    
    print("A área do retângulo = {0} \n".format(area))
    base = float(input("Digite a base do retângulo: "))
print("\nFim do Programa!")

#Exemplo 2
base = 1

while base <= 5:
    altura = 2
    while altura <= 10:
        area = base * altura
        
        print("Base = {0} e altura = {1}".format(base, altura))
        print("Área do retângulo = {0} \n".format(area))
        
        altura += 2
base += 1